core/contracts/financial-templates/implementation/ExpiringMultiPartyCreator.sol

```diff
@@ -11,7 +11,12 @@ import "./ExpiringMultiPartyLib.sol";
 /**
  * @title Expiring Multi Party Contract creator.
  * @notice Factory contract to create and register new instances of expiring multiparty contracts.
```

Comment:
I moved this up and added it to the `notice` comment because it is a general comment around the restrictions in the creator.